axum-login/src/redis_store.rs

```diff
@@ -0,0 +1,81 @@
+use std::marker::{PhantomData, Unpin};
+
+use async_trait::async_trait;
+#[cfg(feature = "redis")]
```

Comment:
I don't think this is needed, since you mark the whole module under this `cfg` [here](https://github.com/maxcountryman/axum-login/pull/50/files#diff-04c7a614937343ceadba57766614d7d6935a4d0dee5e8d5d44066d58d41d419bR188).